manifest.js

```diff
@@ -69,7 +69,8 @@ const manifest = {
             Invite: './server/models/invite',
             Session: './server/models/session',
             Token: './server/models/token',
-            User: './server/models/user'
+            User: './server/models/user',
+            PermissionConfig: './server/models/route-scope'
```

Comment:
Please remove this line of code! as we're not adding database back up in this iteration.